### PR TITLE
feat: enable persistence of function between multiple calls of ExpandedImageForm

### PR DESCRIPTION
**Pull request by nchietala**

### CODE REVIEW

File: OpenMTRDemo/Forms/TiledFiltersForm.cs [hunk 1]
```diff
@@ -37,77 +39,72 @@ public override void loadToolStripMenuItem_Click(object sender, EventArgs e)
...
+            _meterList = new List<KeyValPair<PictureBox>>();
 
-            //OpenCVSharp required Mat data type to not be null when databound.
-            _meterPane1.ModifiedImage = _meter.SourceImage.Clone();
-            _meterPane2.ModifiedImage = _meter.SourceImage.Clone();
-            _meterPane3.ModifiedImage = _meter.SourceImage.Clone();
-            _meterPane4.ModifiedImage = _meter.SourceImage.Clone();
-            _meterPane5.ModifiedImage = _meter.SourceImage.Clone();
-            _meterPane6.ModifiedImage = _meter.SourceImage.Clone();
+            _meterList.Add(new KeyValPair<PictureBox>(pictureBoxSource ,_meter));
```
Comment (cclinard84) [inline OpenMTRDemo/Forms/TiledFiltersForm.cs]: I would probably load the list of key value pairs in a different method than 'LoadAllImagePanes'

File: OpenMTRDemo/Filters/CannyFilter.cs [hunk 2]
```diff
@@ -20,8 +23,7 @@ public override void ApplyFilter(Mat image)
         }
 
         private void threshold_ValueChanged(object sender, System.EventArgs e)
-        {
-            int value;
+        {int value;
```
Comment (cclinard84) [inline OpenMTRDemo/Filters/CannyFilter.cs]: This is rather odd (just the lack of  new line and indentation).

Comment (nchietala) [reply OpenMTRDemo/Filters/CannyFilter.cs]: That was unintentional.

File: OpenMTRDemo/Forms/TiledFiltersForm.cs (hunk 1 above)
Comment (nchietala) [reply OpenMTRDemo/Forms/TiledFiltersForm.cs]: Should I put that into it's own method, then?

**Pull request merged**